app/Http/Kernel.php

```diff
@@ -33,8 +33,9 @@ class Kernel extends HttpKernel
             \Coyote\Http\Middleware\FirewallBlacklist::class
         ],
         'api' => [
-            'throttle:60,1',
-            'bindings'
+            \Illuminate\Session\Middleware\StartSession::class,
+            \Illuminate\Routing\Middleware\SubstituteBindings::class,
+            Middleware\DefaultBindings::class,
```

Comment:
Dobra, z tym throttle to trzeba dodać `'throttle'      => \Illuminate\Routing\Middleware\ThrottleRequests::class,` faktycznie. A co ze StartSession, też dodać czy to inaczej rozwiązać?